components/com_kunena/site/template/crypsis/template.php

```diff
@@ -82,63 +82,20 @@ public function loadLanguage()
 	 */
 	public function initialize()
 	{
-		// Template requires Mootools 1.4+ framework.
-		$this->loadMootools();
```

Comment:
Untill we have not replaced attachments uploader and bbcode editor, we still need to load mootools